schematic/store/synapse.py

```diff
@@ -11,6 +11,7 @@
 import secrets
```

Comment:
nit: I have swapped to using isort to handle ordering imports, that way I never need to think about where to add the import as this will auto sort them for me:

![image](https://github.com/Sage-Bionetworks/schematic/assets/17128019/97e4642f-73c3-4c61-95e1-91ae617e265c)

Reply:
Thanks for the suggestion Bryan! i used isort in the past too. See a separate PR here: https://github.com/Sage-Bionetworks/schematic/pull/1441